.ci/openvino-onnx/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN apt-get update && apt-get -y --no-install-recommends install \
 # OpenVINO dependencies
         build-essential \
         ninja-build \
-        cmake \
+        cmake=3.24.0 \
```

Comment:
it does not work this way, see:
https://github.com/openvinotoolkit/openvino/blob/e8b108ac6b6bffb88d0bee68858276de10904023/install_build_dependencies.sh#L226-L230